documentation/projects/proposals/trust_and_safety/content_report_moderation/20231208-implementation_plan_django_admin_moderator_access.md

```diff
@@ -0,0 +1,491 @@
+  - `duplicate` (marked pending report(s) `no_action` when the work is already
+    marked sensitive; i.e., a re-report that did not result in deindexing)
+
+Additionally, the report models will be expanded with an additional nullable
+column, `decision_id` that ties the report to a specific moderation decision. A
+single moderation decision can happen as a result of multiple pending reports,
+so this facilitates the many-to-one relationship between a report and the
+moderation decision. See
+[the moderation view section below for additional information regarding this](#media-moderation-view).
+
+There will not be a new admin view for `ModerationDecision`. Instead, moderation
+decisions will
+[show up on an individual media item's admin description page](#media-moderation-view).
+The only part of a `ModerationDecision` moderators can manually interact with is
+the explanation on the decision. Everything else is handled by the report and
+media model admins.
+
+```{tip}
+The `action` slugs can be context sensitive, meaning they can take into account
+previous moderation decisions to give better descriptions of what happened. These
+must be selected automatically based on the history of the media and its reports.
+Examples of potential future actions are:
+* `reversed_sensitive`, for reversing a sensitivity designation from a work previously
+  moderated as sensitive
+* `ignore_sensitive_text`, for marking a work to be ignored by sensitive text filtering
+* `reversed_deindex`, for reversing a deindex decision of a work
+* `submitted_metadata_correction`, for some future magical feature where we allow users
+  to submit suggested fixes to the metadata of a work
+
+Again, these are purely hypothetical. They're just meant as examples to drive home the
+_idea_ behind the action column. For now we will only implement the four actions listed
+in the columns list above.
+```
+
+#### Table view improvements
+
+We will scrap the existing report table. Instead of showing a list of reports,
+we'll show a list of reported media, sorted by the total number of pending
+reports, and then by the age of the oldest pending report. This will have the
+effect of sorting re-reported works to the top of the list, under the assumption
+that a work with multiple pending reports is probably higher priority, because a
+decision to report it was taken multiple times. By default, we will only show
+works with "pending" reports, with the option to include other statuses if
+moderators choose. This creates a queue to work through progressively, from top
+to bottom. Remove any manual sorting options for now. We can add those back if
+we get feedback that moderators want them, but for now they just clutter and
+confuse the interface, and make the foundational feature more complex.
+
+#### Media moderation view
+
+Clicking on single report should show the following:
+
+- The details of the reported media:
+  - title
+  - description
+  - tags
+  - the work itself
```

Comment:
Is this the direct URL?

It may also be a given, but it would be good to include in this list the `description` sent with the report (different from the media's description), if any.

Reply:
It would be either the image or audio available to view/play.

I'll make explicit that the report description should be present as well :+1: